### PR TITLE
Kalkulačka - Max

### PR DESCRIPTION
**Pull request by JonathanFortniteKennedy**
Yee 👹👹👹👹👹👹👹

### CODE REVIEW

**Comment by JonathanFortniteKennedy** [comment]
[Kalkulačka](https://trello.com/c/69i48lUJ/44-kalkula%C4%8Dka)

File: src/main/java/org/cakelook/kalkulacka/Main.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            System.out.println("Nejde nulou monkey");
+        }
+        else {
+            System.out.println("Podil " + (knedlik / kvin));
+        }
+    }
+}
+
+
+// 👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹👹
```
Comment (Kostahe) [inline src/main/java/org/cakelook/kalkulacka/Main.java:29]: KVINE MISTO

File: src/main/java/org/cakelook/kalkulacka/Main.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        System.out.println("Rozdil: " + (knedlik - kvin));
+        System.out.println("Soucin: " + (knedlik * kvin));
+
+
+        if (kvin == 0) {
+            System.out.println("Nejde nulou monkey");
+        }
+        else {
+            System.out.println("Podil " + (knedlik / kvin));
+        }
```
Comment (Kostahe) [inline src/main/java/org/cakelook/kalkulacka/Main.java:24]: Crazy stuff

**Review by Kostahe — APPROVED** [review]
Me to nezajima

**Pull request merged**